modules/network-graph-20-view.adoc

```diff
@@ -51,6 +52,20 @@ If you are using offline mode, you can update these ranges by installing new sup
 
 The following image provides an example of the network graph. In this example, based on the options that the user has chosen, the graph depicts deployments in the selected namespace. Traffic flows are not displayed until you click on an item such as a deployment. The graph uses a red badge to indicate deployments that are missing policies and therefore allowing all network traffic.
 
+[id="internal-entities-connections_{context}"]
+== Connections involving internal entities
+
+The network graph is useful for identifying deployments with active connections to entities that do not belong to any known deployment or CIDR block. Some of these connections never reach outside of the cluster and are made within the cluster's private network. The network graph represents those as connections to or from _internal entities_.
+
+Connections with internal entities represent a connection between a deployment and an IP address that belongs to the private address space as defined in RFC 1918. In some cases, Sensor is unable to identify one or both deployments involved in a connection. In that case, the system analyzes the IP address and decides whether the connection is internal or external.
```

Comment:
Not sure if we usually do this, but we could add a link to the RFC: https://datatracker.ietf.org/doc/html/rfc1918

Reply:
We tend to avoid external links because of maintenance but RFCs don't normally change so I think it's fine.